easybuild/base/fancylogger.py

```diff
@@ -580,6 +580,8 @@ def logToFile(filename, enable=True, filehandler=None, name=None, max_bytes=MAX_
         'maxBytes': max_bytes,
         'backupCount': backup_count,
     }
+    if sys.version_info[0] >= 3:
+        handleropts['encoding'] = 'utf-8'
```

Comment:
This deserves a comment to explain why this is needed?

Reply:
Erm, to make sure sure logs are utf-8 encoded? I don't really know what else to explain which isn't already stated by `encoding = utf-8`

All the gory details about default encoding in Python 3, 3.6, 3.7 would IMO be overkill here.

Reply:
But *why* do logs need to be utf-8 encoded all of a sudden :)

Reply:
Because we log potentially UTF-8 encoded strings and if the default doesn't happen to be utf-8 we get encoding errors. This change makes us not rely on the default to be UTF-8 (e.g. on Python 3.6) but choose it explicitly. It accidentally worked prior to the change because we didn't test that case (i.e. no LC_ALL=C), so this is a bugfix
I'll add that to the PR description to answer why we need the change, however I'd avoid adding a more or less trivial comment to the code.

Reply:
OK, fine.